one/src/lib.rs

```diff
@@ -70,6 +70,15 @@ struct DaemonOpts {
     )]
     swarm_addresses: Vec<String>,
 
+    /// Extra bootstrap peer addresses to be used in addition to the official bootstrap addresses.
```

Comment:
Should we add a comment that these address will be a best effort to always remain connected?